packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -375,10 +380,7 @@ export function computeExpirationForFiber(
   return expirationTime;
 }
 
-export function scheduleUpdateOnFiber(
-  fiber: Fiber,
-  expirationTime: ExpirationTime,
-) {
+function scheduleUpdateOnFiber(fiber: Fiber, expirationTime: ExpirationTime) {
```

Comment:
This export was never referenced (only the `scheduleWork` alias below) so I removed it.

Reply:
Sure.